data/json/items/melee/swords_and_blades.json

```
@@ -2572,17 +2572,16 @@
     "type": "TOOL",
     "symbol": "/",
     "color": "light_gray",
-    "name": { "str": "pair of butterfly swords", "str_pl": "pairs of butterfly swords" },
-    "description": "A matched pair of traditional Shaolin butterfly swords.  They are about the same size as machetes, but have hand guards and wider blades.",
-    "//": "basically 2x the weight, 1.3x the damage, 2.5x the resources of a single machete to cover the fact that it's two weapons, each with a hand guard",
+    "name": { "str": "butterfly sword", "str_pl": "butterfly swords" },
```

```suggestion
    "name": { "str": "butterfly sword" },
```

This is unnecessary. The ``s`` is added at the end automatically.